:book: Document KCP objects for different syncer deployment scenarios (#2917)
kcp-dev/kcp: docs/content/concepts/scenarios/README.md

# Document the KCP Workspace objects

> Document key KCP objects, with they Yaml, that are created and modified during the key steps of MC Syncer deployment:

Reviewer: ```suggestion
Document key kcp objects, with example manifests, that are created and modified during the key steps of transparent multi-cluster syncer deployment:

```

> 1. Creation of a KCP Workspace (`k ws create`)

Reviewer: ```suggestion
1. Creation of a KCP Workspace (`kubectl ws create`)
```

nit: prefer not using aliases here - `k` does nothing on my system  and readers may be unfamiliar

> 2. Creation of a KCP Syncer (`k kcp workload sync`)

Reviewer: ```suggestion
2. Creation of a kcp Syncer (`k kcp workload sync`)

```

nit: kcp is not an acronym and is styalized as lower-case

> 3. Deployment of the Syncer in the edge pcluster (`k apply -f`)

Reviewer: Why edge?

We either use "physical cluster" or "workload cluster" as identifiers for the clusters running workloads - let's expand out "pcluster" to one of those to be more specific.

Reviewer 2: Yes, this is a TMC document, let us not confuse it with edge-mc.
